platform/contracts/admin/src/contracts/mod.rs

```diff
@@ -32,7 +32,7 @@ pub type ContractsMigration = ContractsTemplate<Option<MigrationSpec>>;
 
 pub type ContractsPostMigrationExecute = ContractsTemplate<Option<String>>;
 
-pub(crate) type ContractsGroupedByProtocol = ContractsTemplate<Addr, BTreeMap<String, Protocol>>;
+pub type ContractsGroupedByProtocol = ContractsTemplate<Addr, BTreeMap<String, Protocol>>;
```

Comment:
who is gonna use it outside the crate?

Reply:
It is used outside, in the `InstantiateMsg`, if I recall correctly. The compiler was livid about it.